05-geometry-operations.qmd

```diff
@@ -651,14 +660,138 @@ fig, axes = plt.subplots(ncols=2, figsize=(8,4))
 rasterio.plot.show(src.read(1)[:5, :5], transform=src.transform, ax=axes[0])
 rasterio.plot.show(r2[:25, :25], transform=new_transform2, ax=axes[1])
 axes[0].set_title('Original')
-axes[1].set_title('Disaggregated');
+axes[1].set_title('Disaggregated (bilinear)');
 ```
 
 Code to export the disaggregated raster would be identical to the one used above for the aggregated raster, so omit it to save space.
 
 ### Resampling {#sec-raster-resampling}
 
-...
+The above methods of aggregation and disaggregation are only suitable when we want to change the resolution of our raster by the aggregation/disaggregation factor. 
+However, what to do when we have two or more rasters with different resolutions and origins? 
+This is the role of resampling---a process of computing values for new pixel locations. 
+In short, this process takes the values of our original raster and recalculates new values for a target raster with custom resolution and origin (@fig-raster-resample).
+
+There are several methods for estimating values for a raster with different resolutions/origins (@fig-raster-resample). The main resampling methods include:
+
+* Nearest neighbor: assigns the value of the nearest cell of the original raster to the cell of the target one. This is a fast simple technique that is usually suitable for resampling categorical rasters.
+* Bilinear interpolation: assigns a weighted average of the four nearest cells from the original raster to the cell of the target one. This is the fastest method that is appropriate for continuous rasters.
+* Cubic interpolation: uses values of the 16 nearest cells of the original raster to determine the output cell value, applying third-order polynomial functions. Used for continuous rasters and results in a smoother surface compared to bilinear interpolation, but is computationally more demanding.
+* Cubic spline interpolation: also uses values of the 16 nearest cells of the original raster to determine the output cell value, but applies cubic splines (piecewise third-order polynomial functions). Used for continuous rasters.
+* Lanczos windowed sinc resampling: uses values of the 36 nearest cells of the original raster to determine the output cell value. Used for continuous rasters.
+* Additionally, we can use straightforward summary methods, taking into account all pixels that coincide with the target pixel, such as average (@fig-raster-aggregate), minimum, maximum (@fig-raster-resample), median, mode, and sum.
+
+The above explanation highlights that only nearest neighbor resampling is suitable for categorical rasters, while all methods can be used (with different outcomes) for continuous rasters. 
```

Comment:
"while all methods" -> maybe "while all remaining methods" (to highlight not to use nn for continuous variables?)

Reply:
> "while all methods" -> maybe "while all remaining methods" (to highlight not to use nn for continuous variables?)

Right! Thanks